### PR TITLE
Backend testing capability. 

### PR DESCRIPTION
**Pull request by Moo64c**
#173 
And per-issue testing.

### CODE REVIEW

File: productivity/modules/custom/productivity_github/productivity_github.table.inc [hunk 1]
```diff
@@ -0,0 +1,163 @@
+<?php
+/**
+ * @file
+ * Code for the Productivity time tracking feature.
+ */
+define('PRODUCTIVITY_GITHUB_ISSUE_OPEN_STATUS', t('WIP'));
```
Comment (amitaibu) [inline productivity/modules/custom/productivity_github/productivity_github.table.inc]: not doing a review  - just so this. No t() in define.

Comment (Moo64c) [inline productivity/modules/custom/productivity_github/productivity_github.table.inc]: Fixed, thanks

**Comment by Moo64c** [comment]
@bricel Ready for review.

**Comment by Moo64c** [comment]
@bricel This also includes a bug fix.

**Pull request merged**